.github/workflows/code_review.yml

```diff
@@ -12,9 +12,10 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v3
       - name: Code Review
-        uses: freeedcom/ai-codereviewer@main
+        uses: ./
         with:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           OPENAI_API_KEY: ${{ secrets.OPENAI_API_KEY }}
-          OPENAI_API_MODEL: "gpt-4-1106-preview"
+          OPENAI_API_MODEL: "gpt-4o-mini"
           exclude: "yarn.lock,dist/**"
+          sourcemaps: true
```

Comment:
Enabling 'sourcemaps' could enhance the code review process by providing better context for the code being analyzed. Ensure that sourcemaps are correctly generated and accessible during the review.